### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by CurtisGreen** [RiotGames/developer-relations#667]
**Bug Description**  
V5 match list with `type=ranked` contains Clash games

**Problem Description**  
V5 match list with the `type=ranked` filter should only contain solo/duo and flex games

**Expected Result**  
When I pass in `type=ranked` to `/lol/match/v5/matches/by-puuid/{puuid}/ids` I expect ranked solo/duo and ranked flex games

**Actual Result**  
I get ranked solo/duo, ranked flex, and Clash games

**Developer Impact**  
* Frequency: Every API call
* Severity: Low
* Efficiency: Requires either extra match history calls or extra match calls

**Player Impact**  

* Frequency: Every time the user visits the site, they will have slightly longer load times for recent ranked match history due to extra requests
* Region: All

**Issue Comments**  
- type=ranked including clash games is misleading to developers
- The in-client clash scouting page shown before clash games seems to use this filter and gives misleading data to the user by including clash games in the ranked match history tab
- This causes a slight increase in the number of requests as developers either need to query match history for clash games then throw them away, or query match history for ranked solo/duo and flex separately

**Comment by geozukunft** [comment]
This is not a bug but rather intentional it is considered to be within the group of ranked type games as it falls within the category of competitive game modes even if there is no ranked ladder attached to it.

**Comment by CurtisGreen** [comment]
Interesting, if that is the case I still think it’s misleading when looking at how Riot and 3rd party developers mostly use the API. My last two bullet points give good examples of the negative effects of this, do you know of any positive effects that I might be unaware of? 

Thanks

**Comment by DarkIntaqt** [comment]
Note: ARAM clash does not appear within the ranked filter. This leads to further inconsistencies with the query options in match v5. 
Therefore, this issue is maybe not as "unnecessary" as stated before.

**Comment by geozukunft** [comment]
To respond to the last part of the previous response from @CurtisGreen in the end you still need the clash + ranked games and if you download the games themself you will know which game it was exactly anyway. And I'd say that the low quantity of Clash games would not warrant giving it a seperate filter option that would be my best guess on why Riot decided to structure it this way.

**Comment by CurtisGreen** [comment]
Yeah I don’t think clash should have its own filter unless they start doing many different kinds of clash like ARAM clash which have different queue IDs. I just think removing clash from the ranked filter would benefit both Riot and 3rd party devs. 

This is obviously opinionated, so to start off I would just like to make sure this is intentional by Riot. If it is intentional and they still want it to be like that then so be it

**Comment by geozukunft** [comment]
Maybe @RiotTuxedo can give us some insight as he worked on it afaik.
